lib/fuzzy-finder-view.js

```diff
@@ -44,7 +44,7 @@ export default class FuzzyFinderView {
           this.previousQueryWasLineJump = true
           this.selectListView.update({
             items: [],
-            emptyMessage: 'Jump to line in active editor'
+            emptyMessage: 'Enter a <row> or <row>:<column> to go there. Examples: "3" for row 3 or "2:7" for row 2 and column 7'
```

Comment:
I just copied it from the existing go-to-line package as I was intending to follow this PR with changes to `ctrl-g` to use a command from this package so as to deprecate the `go-to-line` package.

I agree that's a terribly long message. 

Should I still change it?